src/Commands/Concerns/InteractsWithDDD.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Commands\Concerns;
+
+use Illuminate\Support\Str;
+
+/**
+ * @deprecated
+ */
+trait InteractsWithDDD
+{
+    protected function resolvePathForStructure(string $key): string
+    {
+        /** @var string $path */
+        $path = config(key: 'diamond.structures.' . $key);
+
+        return $path;
+    }
+
+    protected function resolveBasePath(): string
+    {
+        /** @var string $path */
+        $path = config(key: 'diamond.base_directory');
+
+        return $path;
+    }
+
+    protected function resolveDomainPath(): string
+    {
+        return $this->resolvePathForStructure(key: 'domain');
+    }
+
+    protected function resolveInfrastructurePath(): string
+    {
+        return $this->resolvePathForStructure(key: 'infrastructure');
+    }
+
+    public function resolveNamespace(string $structures, string $suffix, string $prefix): string
+    {
+        return Str::of(string: '\\:prefix\\:suffix')
+            ->start($structures)
+            ->replace(search: ':prefix', replace: $prefix)
+            ->replace(search: ':suffix', replace: $suffix);
+    }
+
+    public function resolveNamespacePath(string $namespace): string
+    {
+        return base_path(
+            path: $this->resolveBasePath() . Str::replace(
+                search: '\\',
+                replace: '/',
+                subject: $namespace
+            )
+        );
+    }
+
+    protected function resolveNameFromPhp(string $name): string
```

Comment:
what if this function use uppercase for Php, like`resolveNameFromPHP`, and the others are changed like that too

Reply:
Sure